### DISCUSSION TITLE
Exporting list of Pimcore Users

### DISCUSSION THREAD

**Discussion by cancan101** [pimcore/pimcore#17668] [Q&A / Support] [answered]
What is the easiest way to export a list of users in the Pimcore system (including name, email address, active/not), etc? I know I can dump this from the DB but wanted to see if there is anything exposed in the UI.

**Comment by idna001** [comment, marked as answer]
Hi, 
there is no admin user export feature in the UI. But you can easily dump the affected tables. 

Including: 
	1. users
	2. users_permissions_defintions
	3. users_workspaces_asset
	4. users_workspaces_document
	5. users_workspaces_object

Although this would be a useful new feature. Maybe you want to raise an issue.

Andreas

**Reply by cancan101** [reply]
Posted: https://github.com/pimcore/pimcore/issues/17718
